history/ChaosDAOHistory.md

```diff
@@ -0,0 +1,24 @@
+
+In August 2021, there was excitement around the launch of the Karura (Acala) dex parachain, and then the Moonriver EVM parachain. This launched a frenzy of defi degeneracy, especially around the Solarbeam dex and the Moon Kafe auto-compounding service.
+
+In late November 2021, ChaosDAO members joined together to fund a Moonriver collator as a collective investment but also as the first constant revenue stream to create a ChaosDAO treasury. CD members benefited from the discounted node operation services of Mayhem Nodes, advice from Alex Hartoum with Polkahaus, and whale stake from Donnie Big Bags to help us stay in the active set during our first experiences surviving “turbulence” of MOVR nodes competing to stay in the active set. Returns were estimated at 130%, with 2% going to Mayhem as the node operator, 10% to the ChaosDAO treasury, and the remaining 88% of collator revenue being paid out to the 12 contributors to the collator.
+
+Also in November 2021, a group of OG “Ohmies” decided to fork the OHM project over to the Moonriver ecosystem and naming it “Rome”. Their objective was to first use the OHM codebase to start a reserve currency and then create a game to use the project token as the in-game currency. Rome organizers announced that people could organize themselves into Houses which would then govern through a Senate of elected representatives. Several members of ChaosDAO had already been interested in OHM and were excited to help with the launch of an OHM fork in the Dotsama ecosystem and therefore decided to petition to create one of the Houses of Rome which they named “House of Chaos”. Initial admins were all from ChaosDAO, but general membership was open and public. ChaosDAO members remained engaged with the Rome project, especially Kratistos as Moderator and bacchist as one of the Scholars of the Bank of Rome. Unfortunately, the Rome development team lost momentum trying to do do complex 3D graphics suited to use on a mobile phone and the Rome token price fell below the value of its treasury. Eventually vultures from elsewhere bought up Rome tokens and launched a successful governance attack because the distributed House→Senate→Snapshot governance vote pathway had never been finalized and therefore a simple Snapshot vote was able to be won by the new token holders in order to end the project and disburse current treasury holdings to token holders.
+
+In Spring 2022, several ChaosDAO members were chatting in the #gm channel of ChaosDAO and came up with the idea to create a cheap Kusama parachain which would allow users to send and receive GM / GN messages, but whose ultimate intent was to be a test ground for lots of crazy ideas not tested by other parachain teams.
```

Comment:
Last line I might change the intent part to "be a playground" only because you have "ideas not tested" (so as to avoid using test twice; alternatively, you could keep test ground and swap tested for "not attempted")

Reply:
hmm `playground` seems perhaps too dismissive (children play on playground)
but good catch on the two `test` references. Changed to:
`to be a testing ground for lots of crazy ideas not pursued by other parachain teams.`